### ISSUE TITLE
CP/M version of the tests

### ISSUE THREAD

**Issue by kosarev** [raxoft/z80test#1]
Hi Patrik, is there a CP/M version of these tests? If not, would you be interested in patches supporting that? Thanks.

**Comment by raxoft** [comment]
IIRC someone adapted it for some CPC machine, but I don't know if it was for the CP/M system or not. In either case, feel free to fork it and create your version. However, just know that I do not have time to maintain versions for other systems. Thanks.

**Comment by EdouardBERGE** [comment]
Hi, not for CPM but for Amstrad, i think there is no more than rewrite "printchr" to CALL #BB5A (keeping your PUSH/POP around as well) and it's ok but...
i downloaded last sjasm assembler which seems not to please your source code
What version do you use?
And BTW is it possible to have a targer only with binary output?
I tried "sjasm z80doc.asm" without success (MANY error messages, on macro params, ...)

**Comment by EdouardBERGE** [comment]
i compiled sjasmplus 1.20.1 to try, it's better but still MANY errors

**Comment by EdouardBERGE** [comment]
wouhou, i succeed downloading sjasm0.42c for windows in a windows virtual machine ^_^
sorry for disturbance but i think you should warning about sjasm ecosystem (BTW sjasm linux cannot be compiled :/ )

**Comment by raxoft** [comment]
Yes, I used the sjasm0.42c from http://www.xl2s.tk/. The linux version worked for me too last time I tried, but I might have tweaked a thing or two...
